test/test_db.cpp

```diff
@@ -7,11 +7,13 @@
 
 namespace xeus_sqlite
 {
+interpreter inter();
```

Comment:
It's better to create your instance in your test. This will prevent side effects between tests (one test causing another one to fail, because of the interpreter state). Tests should be independent.

Reply:
Oh, okay. Makes sense :)